.changeset/gorgeous-bears-glow.md

```diff
@@ -0,0 +1,57 @@
+---
+'braid-design-system': major
+---
+
+Improved trimming of white space around text
+
+
+## Breaking Changes
```

Comment:
I don't think we can use headings here since this ultimately gets nested inside a bullet list. This is why I usually go for the bold + all-caps style.